src/components/organisms/UserProfileModal/UserProfileModal.tsx

```diff
@@ -83,7 +90,7 @@ const UserProfileModal: React.FunctionComponent<PropTypes> = ({
               </div>
               <div className="profile-text">
                 <h2 className="italic">
-                  {userProfile.partyName || "Captain Party"}
+                  {selectedUserProfile.partyName || "Captain Party"}
                 </h2>
```

Comment:
`"Captain Party"` should be moved to a const in `src/settings.ts` (as something like `DEFAULT_PARTY_NAME`) and then referenced here.

(I feel like we already made this change somewhere.. maybe it's in one of @denisdimitrov's outstanding branches or similar?)